cli/commands/project.go

```diff
@@ -8,7 +8,7 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/phase2/rig/cli/commands/project"
+  "github.com/phase2/rig/cli/commands/project"
```

Comment:
Looks like a tab vs/ spaces issue. Please tell me you are setup to use spaces? 😄